workflow/rules/picard_collect_multiple_metrics.smk

```diff
@@ -0,0 +1,41 @@
+__author__ = "Martin Rippin"
+__copyright__ = "Copyright 2021, Martin Rippin"
+__email__ = "martin.rippin@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule picard_collect_multiple_metrics:
+    input:
+        bam="alignment/merge_bam/{sample}_{type}.bam",
+        ref=config["reference"]["fasta"],
+    output:
+        expand(
+            "qc/picard_collect_multiple_metrics/{{sample}}_{{type}}.{ext}",
+            ext=config.get("picard_collect_multiple_metrics", {}).get("output_ext", ""),
```

Comment:
should we maybe set a default value here? config schema doesn't say that output_ext is required.

Reply:
I don't think we need to? If you don't want to run `picard_collect_multiple_metrics` you can ignore this stanza and it will not produce any output for this rule (in the `all` rule the loop is not initiated). If we require it, then you have to set the stanza even though you might not want any output from it. Dunno. I think, as long as it's handled either way, it is better to leave it optional.